ui/components/app/signature-request-original/index.scss

```diff
@@ -19,9 +19,7 @@
       box-shadow: none;
     }
 
-    @include screen-sm-min {
-      height: 650px;
-    }
+    @include screen-sm-min;
```

Comment:
We cam remove this mixin all together it just puts any styles inside a media query so if there are no styles it's not required.
```suggestion
```

Reply:
ok, done - thanks

Let me know if there are any other tickets you want a hand with we're while waiting for another approval